recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>People do not know in general the 'par' format. Maybe you could say that this is described in section 2.2... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>